### PR TITLE
feat: [rhel_for_edge] Update docstring to make it more readable

### PR DESCRIPTION
**Pull request by psachin**
This is a follow-up of https://github.com/RedHatInsights/insights-core/pull/3526 to make the docstring more readable.

Signed-off-by: Sachin Patil <psachin@redhat.com>

### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [X] Is this PR an enhancement?

### CODE REVIEW

File: insights/combiners/rhel_for_edge.py [hunk 1]
```diff
@@ -17,34 +17,36 @@
...
-    Edge computing system packages are managed via rpm-ostree.
-    Red Hat CoreOS is also managed via rpm-ostree, use the string "Red Hat Enterprise Linux release"
-    from "/etc/redhat-release" to determine if it is an edge computing system as it is
-    "Red Hat Enterprise Linux CoreOS release" on RedHat CoreOs.
+    """Combiner for checking if the system is an edge computing system. Edge
+    computing as well as the Red Hat CoreOS packages are managed via rpm-ostree.
+    Use the string "Red Hat Enterprise Linux release" from
+    ``/etc/redhat-release`` to determine an edge computing system. The Red Hat
+    CoreOS system will have "Red Hat Enterprise Linux CoreOS release" as the
+    string..
```
Comment (jobselko) [inline insights/combiners/rhel_for_edge.py:25]: Double `.`

File: insights/combiners/rhel_for_edge.py [hunk 2]
```diff
@@ -17,34 +17,36 @@
...
+    When an edge computing system (created from online console edge image) is
+    configured to use the automated management, the output of ``rhc status`` is
+    as below::
 
-            Connection status for test.localhost:
+            Connection status for <HOST>:
             - Connected to Red Hat Subscription Management
             - The Red Hat connector daemon is active
 
-    If a system can upload insights archive, it must be connected to Red Hat Subscription Management,
```
Comment (jobselko) [inline insights/combiners/rhel_for_edge.py:37]: You omitted this part. I am unsure if it is important to have it here. What do you think?

**Review by jobselko — APPROVED** [review]
Looks good!

File: insights/combiners/rhel_for_edge.py (hunk 1 above)
Comment (psachin) [reply insights/combiners/rhel_for_edge.py:25]: Oops! Corrected.

File: insights/combiners/rhel_for_edge.py (hunk 2 above)
Comment (psachin) [reply insights/combiners/rhel_for_edge.py:37]: In my opinion the line was not "connecting" well with the subsequent line. Also it is obvious that the system should be subscribed & registered with Red Hat in order to upload an insights archive.

Comment (jobselko) [reply insights/combiners/rhel_for_edge.py:37]: That seems reasonable. I agree with you.

**Pull request merged**